### ISSUE TITLE
spglib compilation on matgen

### ISSUE THREAD

**Issue by matk86** [materialsproject/MPenv#8]
while setting up the virtual env on matgen i noticed that the default intel compiler fails to compile the spglib dependency required by pymatgen properly. Switching to gcc(export CC=gcc) solves the problem.

**Comment by tschaume** [comment]
That's to be expected as mentioned in Part 4 Bullet #4 in the README. It's
admittedly somewhat hidden, though ;-)

**Comment by matk86** [comment]
ok. thanks

**Comment by dwinston** [comment]
I found it wasn't enough to simply unload the intel module. I had to `export CC=gcc` as @matk86 said. I also had to add an explicit `pip install spglib` (e.g. inserted at https://github.com/materialsproject/MPenv/blob/8345d54cfa7ca2b7e114c5a3790f492ebfca92f9/MPenv/mpenv.py#L107) to get a fresh `mpenv <MY_ENV>` to complete successfully. Since this was necessary for a fresh install, in might be good to note this earlier than Part 4 in the README.
